internal/v1/handler_blueprints.go

```diff
@@ -176,6 +178,12 @@ func (h *Handlers) UpdateBlueprint(ctx echo.Context, blueprintId uuid.UUID) erro
 }
 
 func (h *Handlers) ComposeBlueprint(ctx echo.Context, id openapi_types.UUID) error {
+	var requestBody ComposeBlueprintJSONBody
+	err := ctx.Bind(&requestBody)
```

Comment:
I'm not sure if we can check for empty body, it would be nice to be able to send empty body and not fail :)

Reply:
There must be a test which sends no body (body length is 0), otherwise we are breaking the API thus current UI.

Reply:
added test that checjk it